cpp/include/cugraph/detail/graph_functions.cuh

```diff
@@ -158,35 +158,63 @@ partition_information(raft::handle_t const& handle, GraphViewType const& graph_v
  * Collect all the edges that are present in the adjacency lists on the current gpu
  *
  * @tparam GraphViewType Type of the passed non-owning graph object.
- * @tparam EdgeIndexIterator Type of the iterator for edge indices.
  * @tparam GPUIdIterator  Type of the iterator for gpu id identifiers.
- * @param handle RAFT handle object to encapsulate resources (e.g. CUDA stream, communicator, and
- * handles to various CUDA libraries) to run graph algorithms.
- * @param graph_view Non-owning graph object.
- * @param active_majors_in_row Device vector containing all the vertex id that are processed by
+ * @param[in] handle RAFT handle object to encapsulate resources (e.g. CUDA stream, communicator,
+ * and handles to various CUDA libraries) to run graph algorithms.
+ * @param[in] graph_view Non-owning graph object.
+ * @param[in] active_majors_in_row Device vector containing all the vertex id that are processed by
```

Comment:
We're trying to avoid mixing row/column & src/dst and consistently use src/dst (at least in our public API, but I assume it will be nice to avoid mixing in other places as well). And this function really needs to know whether source is major or minor? If not, better name this as just `active_majors`.

Reply:
active_majors_in_row implies all the majors that can be represented by the gpus that have the same row_rank as the current one. The row over here refers to the row/col nomenclature in the comms as opposed to src/dst.

Reply:
Gotcha, then, can't we name this as something like `active_majors_in_same_row_rank`? And could you double check the comment here? This says "all the vertex id that are processed by gpus in the **column** communicator"